Mage.Sets/src/mage/cards/j/JaceTelepathUnbound.java

```diff
@@ -74,104 +63,3 @@ public JaceTelepathUnbound copy() {
         return new JaceTelepathUnbound(this);
     }
 }
-
-class JaceTelepathUnboundEffect extends OneShotEffect {
```

Comment:
I refactored this into CastCardFromGraveyardEffect so it could be used here and on the Chandra PW